lib/Core/loadJson5.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
+/* global require  */
+
+var json5 = require('json5');
+var loadText = require('terriajs-cesium/Source/Core/loadText');
+var defined = require('terriajs-cesium/Source/Core/defined');
+var clone = require('terriajs-cesium/Source/Core/clone');
+
+var defaultHeaders = {
+    Accept : 'application/json,*/*;q=0.01'
```

Comment:
there's no official json5 mimetype, but we should probably pre-empt `application/json5`